cgi-bin/LJ/Talk.pm

```diff
@@ -1830,7 +1831,8 @@ sub talkform {
 
             $ret .= "<input type='hidden' name='cookieuser' value='$remote->{'user'}' id='cookieuser' />\n";
             if ($screening eq 'A' ||
-                ($screening eq 'F' && !$journalu->trusts_or_has_member( $remote ))) {
+                ($screening eq 'F' && !$journalu->trusts_or_has_member( $remote )) ||
+                ( $journalu->has_autoscreen( $remote ) )) {
```

Comment:
Thanks for double checking.  I feel like that logic could be cleaned up to be less confusing, but it can wait for another day.